kong/templates/kong_defaults.lua

```diff
@@ -26,6 +26,7 @@ cluster_mtls = shared
 cluster_ca_cert = NONE
 cluster_server_name = NONE
 cluster_data_plane_purge_delay = 1209600
+cluster_version_check = exact
```

Comment:
What is the reasoning behind this default? This makes upgrades a lot harder. `major_minor` seems a sane default for most cases.